shared/middleware/net_sim_api.rb

```diff
@@ -504,7 +504,7 @@ def delete_messages_for_nodes(shard_id, node_ids)
   # @param [String] event_type - unique metric key within NetSimApi
   # @param [Number] value (default 1) value of measurement, omit if we only care
   #   about event counts.
-  def record_metric(event_type, value = 1)
+  def record_metric(event_type, value = 1, unit = 'Count')
```

Comment:
super small - I would add a param comment for the new param, just because we have them for the others